genome_designer/main/migrations/0005_auto__add_field_referencegenome_metadata.py

```diff
@@ -10,13 +10,14 @@ class Migration(SchemaMigration):
     def forwards(self, orm):
         # Adding field 'ReferenceGenome.metadata'
         db.add_column(u'main_referencegenome', 'metadata',
-                      self.gf('main.custom_fields.PostgresJsonField')(default='{"is_from_de_novo_assembly": false}'),
+                      self.gf('main.custom_fields.PostgresJsonField')(default='{}'),
```

Comment:
These changes to auto-generated migration are a bit sketch. Are you sure they are correct?